src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -161,6 +161,7 @@ void PacketDataReporter::OnCommissionableNodeSrvRecord(SerializedQNameIterator n
     {
         strncpy(mDiscoveredNodeData.instanceName, name.Value(), sizeof(DiscoveredNodeData::instanceName));
     }
+    mDiscoveredNodeData.port = srv.GetPort();
```

Comment:
Need to store the interface id somewhere in this general code too.

Reply:
Done.